src/coreclr/pal/src/init/pal.cpp

```diff
@@ -719,13 +719,15 @@ PAL_InitializeCoreCLR(const char *szExePath, BOOL runningInExe)
         return ERROR_SUCCESS;
     }
 
+#ifndef __wasm__ // we don't use shared libraries on wasm
```

Comment:
```suggestion
#ifndef TARGET_WASM // we don't use shared libraries on wasm
```
Use upper case ifdefs for consistency?

(I am not commenting on all of them.)